js/components/Leaderboard/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { map } from 'lodash';
+
+import { RestApiClient } from '@dosomething/gateway';
+import Table from '../Table';
+
+class Leaderboard extends React.Component {
+	constructor() {
+		super();
+
+		this.state = {
+			jsonReturnedValue: []
+		}
+
+		this.api = new RestApiClient('http://rogue.test/');
+	}
+
+	componentDidMount() {
+		this.api.get('api/v3/signups', {
+			filter: {
+				campaign_id: '1283',
```

Comment:
Is this the right campaign id? (just making sure it is this in prod)

Reply:
oh right! This was just a random one - I'll swap out for the one in prod!

Reply:
this campaign isn't created yet - we'll have to swap this out when it goes live!

Reply:
@chloealee cool! just double checking - is it definitely a prod campaign id (vs something on staging or thor)?

Reply:
@ngjo gave me last year's `campaign_run_id` (7536) so I'll swap that out as a placeholder until this year's campaign is created! @sbsmith86